src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -298,30 +301,35 @@ private boolean isTerminated(final DetailAST ast, boolean useBreak,
                 terminated = true;
                 break;
             case TokenTypes.LITERAL_BREAK:
-                terminated = useBreak;
+                terminated = useBreak
+                    || ast.getChildCount() == 2 && !labels.contains(ast.getFirstChild().getText());
                 break;
             case TokenTypes.LITERAL_CONTINUE:
                 terminated = useContinue;
                 break;
```

Comment:
What about a continue label? We don't need similar label tracking?

Reply:
yes, it needs. Fixed

Reply:
Unless I am missing it, I am not seeing new test inputs for `continue <label>`. Please add some and also please regenerate regression. Will be interesting if anything new for `continue` comes up.

Reply:
I dont see much difference between `break` and `comtinue`, so imo same file is ok. I will rename it.
I will create new report as well, but I really doubt there will be something - continue with break is even less frequent than labelled break.

Reply:
done